ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/ethernet-interfaces.j2

```diff
@@ -58,7 +58,10 @@ ethernet_interfaces:
 {%                     endif %}
     mode: {{ adapter_settings.mode | arista.avd.default }}
     vlans: {{ adapter_settings.vlans | arista.avd.default }}
-{%                     if adapter_settings.native_vlan is arista.avd.defined %}
+{%                     if adapter_settings.native_vlan_tag is arista.avd.defined %}
+    native_vlan_tag: adapter_settings.native_vlan_tag
+{%                     endif %}
+{%                     if adapter_settings.native_vlan is arista.avd.defined(fail_action='warning', var_type='int') %}
```

Comment:
I don't think we need to introduce type check here. Someone could have `"123"` which has been accepted until now. We will add warning to type checks in the upcoming schema validation.

Reply:
so are you saying that if I put warning the configuration would not be rendered then? I thought warning meant "yeah ok this is accepted but beware"